app/views/saml_post/auth.html.erb

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
```

Comment:
Not sure if we want to extract the shared layout here to avoid duplication... I could go either way.

Reply:
I think we can use the `render_template_for` helper in the `SamlIdpController` as a guide for rendering the template. A lot of the tooling for auto-posting should already exist there.

Reply:
@jmhooper the issue is that the structure of the current template (from `SamlIdpController` is a little different from the way I set things up (since we also need to potentially include the signature). That said, I could probably refactor the original template instead of setting up a new one.

Reply:
Sorry, not suggesting we use the same template. Just the same pattern. I think explicitly separate templates is actually a better approach here. I misunderstand the comment earlier. And thought the question was about whether the form should be rendered in a layout. I think this is a place where we should embrace duplication.

Reply:
Cool, so we should just leave this as-is then?

Reply:
yep